detection_rules/rule_validators.py

```diff
@@ -550,4 +592,4 @@ def extract_error_field(source: str, exc: Union[eql.EqlParseError, kql.KqlParseE
     line = lines[exc.line + mod]
     start = exc.column
     stop = start + len(exc.caret.strip())
-    return line[start:stop]
+    return re.sub(r'^\W+|\W+$', '', line[start:stop])
```

Comment:
I dont recall. What was this for?

Reply:
This is to prevent quotes or non alpha numeric charachters from being considered a field. Without this change the returned value could be `"kibana.alert.rule.tags"` instead of `kibana.alert.rule.tags`, which would cause issues if we directly depended on this value to be the field name.